components/steplist/stories/steplist.stories.js

```diff
@@ -50,14 +52,68 @@ export default {
 		isSmall: false,
 		isInteractive: false,
 		withTooltip: false,
+		items: [
+			{
+				isComplete: true,
+			},
+			{
+				isComplete: true,
+			},
+			{
+				isSelected: true,
+			},
+			{
+			},
+		],
```

Comment:
I just rebased, and have labels now: 

<img width="464" alt="Screenshot 2024-07-22 at 11 34 00 AM" src="https://github.com/user-attachments/assets/988380da-674d-4410-bbf1-ffd2a06d9d43">

I probably build Default that way at first since that's how the open source docs page is: 

<img width="500" alt="Screenshot 2024-07-22 at 11 34 35 AM" src="https://github.com/user-attachments/assets/a33b0755-57ec-4c3b-aaa3-d9d79f3b3c64">

Those labels are in `items`, and `items` is disabled in the controls table, so I haven't changed it yet. I'm open to bringing that up if you think we should!

Reply:
Maybe I don't even need the `WithLabel` story, now that `Default` has labels. There's also an `isSmall` control, which removes the labels/tooltips. Let me know what you think, otherwise I can drop this commit.

https://github.com/adobe/spectrum-css/pull/2900/commits/24b89cc159df0a16e687ebf7073ec81d25085ec7